patch/files/scripts/outside.gd

```diff
@@ -2834,15 +2898,184 @@ func _on_capturedmindread_pressed():
 func _on_quicksell_pressed():
 	var text = ''
 	var gold = 0
-	var array = globals.state.capturedgroup
-	globals.state.capturedgroup = []
+	var array = []
+	#ralph5
+	var racepricemodchange = 0
+	var racepricemod = 1
+	#/ralph5
+	for i in globals.state.capturedgroup:
+		array.append(i)
 	for i in array:
-		gold += i.sellprice()/2
+		gold += i.sellprice()*.9 #ralph3

```

Comment:
array and capturedgroup regressed, sellprice modified without condition

Reply:
Fixed